qulice-checkstyle/src/main/java/com/qulice/checkstyle/CheckstyleValidator.java

```diff
@@ -112,10 +112,17 @@ public void validate(final Environment env) throws ValidationException {
      * @see #validate()
      */
     private Configuration configuration(final Environment env) {
+        final File cacheFile =
+                new File(env.tempdir(), "checkstyle/checkstyle.cache");
+        if (!cacheFile.getParentFile().mkdirs()) {
+            throw new IllegalStateException("Unable to crate directories"
+                    + " needed for " + cacheFile.getPath()
+            );
+        }
         final Properties props = new Properties();
         props.setProperty(
             "cache.file",
-            new File(env.tempdir(), "checkstyle/checkstyle.cache").getPath()
```

Comment:
Checkstyle has never been able to create directory structure for cache file, but previously exception was swallowed: https://github.com/checkstyle/checkstyle/commit/ea9ff3965d9c657b5b7bd2d969e075f430635c47#diff-08bcb3012a3f44d264098cc63eabb365L153